coretex/cli/config.py

```diff
@@ -0,0 +1,193 @@
+    with open(dockerScript, "w") as dockerFile:
+        dockerFile.write(dockerScriptContent)
+
+    os.chmod(dockerScript, 0o755)
+
+
+def generateDockerCompose(
+    name: str,
+    memory: int,
+    swap: int,
+    shm: int,
+    organizationId: str
+) -> None:
+
+    dockerComposeContent = f"""version: "3.8"
+services:
+  coretex-node-service:
+    image: {SERVER_URL}:{NODE_PORT}/coretex-node-dev:latest
+    volumes:
+      - {Path.home()}/.coretex:/root/.coretex
+    environment:
+      - CTX_NODE_NAME={name}
+      - CTX_API_URL=https://devext.biomechservices.com:29007/
+      - CTX_STORAGE_PATH=/root/.coretex
+      - CTX_ORGANIZATION_ID={organizationId}
+    deploy:
+      replicas: 1
+      update_config:
+        order: start-first
+    ports:
+      - 21000:21000
+    extra_hosts:
+      - dev.biomechservices.com:192.168.135.10
+    restart: always
+    mem_limit: {memory}g
+    memswap_limit: {swap}g
+    shm_size: {shm}gb
+    cap_add:
+      - SYS_PTRACE
+networks:
+  default:
+    name: coretex_node
+    """
+
+    dockerCompose = CORETEX_CONFIG_DIR / "docker-compose.yml"
+    with open(dockerCompose, "w") as dockerFile:
+        dockerFile.write(dockerComposeContent)
+
+
+def configNode() -> None:
+    config = loadConfig()
+    if not isUserConfigured(config):
+        click.echo("User not configured. Run \'coretex config --user\'", err = True)
+        return
+
+    if isNodeConfigured(config):
+        headers = ["node name", "server", "storage path", "node image"]
+
+        click.echo(click.style("Current Configuration:", fg = "blue"))
+        click.echo(tabulate([[
+            config.get("nodeName"),
+            config.get("serverUrl"),
+            config.get("storagePath"),
+            config.get("image")
+        ]], headers = headers))
+
+        if not click.prompt(
+            "Node configuration already exists. Would you like to update (Y/n)?",
+            type = bool,
+            default = True,
+            show_default = False
+        ):
+            return
+
+    click.echo("[Setting up coretex environment]")
+    click.echo("[Node Configuration]")
+
+    nodeName = click.prompt("Machine name", type = str)
+    organizationId = click.prompt("Organization ID", type = str)
+
+    click.echo("Storage path should be the same as (if) used during --user config")
+    storagePath = click.prompt("Storage path (press enter to use default)", Path.home() / ".coretex", type = str)
+
+    image = click.prompt("Select an image to use for Coretex Node", type = click.Choice(["gpu", "cpu"]), show_choices = True)
+
+    ram = click.prompt("Node RAM memory limit in GB (press enter to use default)", type = int, default = 15)
+    swap = click.prompt("Node swap memory limit in GB, make sure it is larger then mem limit (press enter to use default)", type = int, default = 15)
+    shm = click.prompt("Node shm limit in GB (press enter to use default)", type = int, default = 9)
```

Comment:
Rename to `sharedMemory`. Message should print out `POSIX shared memory` instead of `shm`. Default should be 2 GB